org.sonarlint.eclipse.m2e/src/org/sonarlint/eclipse/m2e/internal/M2eUtils.java

```diff
@@ -65,9 +65,7 @@ private static IFile toSpecificFile(IFile file) {
     IFile finalFile = file;
     IPath rawLocation = file.getRawLocation();
     if (rawLocation != null) {
-      // TODO use getFileForLocation when we will support only Eclipse 4.6+
-      // IFile moreSpecific = ResourcesPlugin.getWorkspace().getRoot().getFileForLocation(rawLocation);
-      IFile moreSpecific = resourceForLocation(rawLocation);
+      IFile moreSpecific = ResourcesPlugin.getWorkspace().getRoot().getFileForLocation(rawLocation);
```

Comment:
Now you should be able to delete method `resourceForLocation` no?

Reply:
Yes. And method `resourceFor`. Removed both.

Reply:
:+1: